web/Makefile

```diff
@@ -24,28 +24,25 @@ changelog: $(CHANGELOG_OUTPUT)
 $(CHANGELOG_OUTPUT): $(CHANGELOG_INPUT)
 	cp -f $< $@
 
-.github:
-	git clone https://github.com/tenzir/.github.git
-
```

Comment:
the reason for switching to submodules was that this fails when you have an empty `.github` directory, which was an issue in an earlier version of this PR because I was mounting a volume on this directory to persist it which was creating the directory. It is not the case anymore. 

I think @dominiklohmann mentioned we can remove it altogether, I personally don't have a strong opinion about it.

Reply:
I'm not saying "no submodule" but rather using the `.github` target to make sure that the submodule is actually checked out.

Reply:
This is actively harmful. The Makefile should not require you to be in a Git context. Also, If I make a local change to the submodule then it should not be undone by the Makefile.

Reply:
oh no, I just reverted it 😄

Reply:
> This is actively harmful. The Makefile should not require you to be in a Git context.

Can you explain why?

> Also, If I make a local change to the submodule then it should not be undone by the Makefile.

This is not a valid use case. The .github directory is an implementation detail from the perspective of the website.

Reply:
It is because building should not require network access, and even less so require Git to be available. If we ever want to integrate this into our Nix environment we'll have a really hard time with it.

I don't think a build process should ever update submodules.

Reply:
I've discussed this with @dominiklohmann. If a user will ever complain about the DX and obscure error messages from a low-level context, he's going to fix it. 🫠

Reply:
I'll re-revert it then